bdk-ffi/src/lib.rs

```diff
@@ -1,15 +1,15 @@
-mod bitcoin;
-mod descriptor;
-mod electrum;
-mod error;
-mod esplora;
-mod keys;
-mod kyoto;
-mod macros;
-mod store;
-mod tx_builder;
-mod types;
-mod wallet;
+pub mod bitcoin;
+pub mod descriptor;
+pub mod electrum;
+pub mod error;
+pub mod esplora;
+pub mod keys;
+pub mod kyoto;
+pub mod macros;
+pub mod store;
+pub mod tx_builder;
+pub mod types;
+pub mod wallet;
```

Comment:
Is there a way to keep the modules private and instead expose just what the new integration tests need with a cfg(test) re-export or similar

Reply:
I think the question things are hingeing on for me is if we are definitely wanting everything to be public/stable; because I think switching these to pub mod makes every existing pub item under those modules part of the crate’s public API (which I think would mean we couldn't rename types or reorganize modules without a breaking change, etc?)... could totally be wrong in how I'm conceiving of it, and I'm not necessarily for/against, mores it's just the main thing I'm thinking thru.

Reply:
yeah at first glance I think this sidesteps that issue nicely! good solution.

let me know if this pr has any more changes coming or not, when I know its finalized again I'll do another close review again

Reply:
Nothing more in this pr. Ready for final review.